packages/react-composites/tests/browser/chat/hermetic/Messaging.test.ts

```diff
@@ -59,24 +59,22 @@ test.describe('Tests related to messaging', async () => {
   test('Inline Image should show a broken image icon when image fetch failed', async ({ page, serverUrl }) => {
     // Mock the api call before navigating
     await page.route(serverUrl + '/images/inlineImageExample1.png', async (route) => {
-      try {
-        await route.fulfill({ status: 300, contentType: 'text/html' });
-      } catch (error) {
-        console.error('Failed at fulfill on route, Error: ', error);
-      }
+      await route.fulfill({
+        status: 404
+      });
     });
 
     await waitForChatCompositeToLoad(page);
 
     await page.goto(
       buildUrlForChatAppUsingFakeAdapter(serverUrl, {
         localParticipant: TEST_PARTICIPANTS[1],
-        remoteParticipants: [TEST_PARTICIPANTS[0], TEST_PARTICIPANTS[2]],
+        remoteParticipants: [TEST_PARTICIPANTS[0], TEST_PARTICIPANTS[1], TEST_PARTICIPANTS[2]],
```

Comment:
Why did we add  `TEST_PARTICIPANTS[1]` to remote participant when `TEST_PARTICIPANTS[1]` is already a local participant?

Reply:
changes are reverted back, i was debugging an issue where message being shown on the left and didn't remove my changes